OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -448,7 +448,8 @@ Action InitInputPanel(Widget panel)
 					{ "CycleStatusBarsKey", "Cycle status bars display" },
 					{ "TogglePixelDoubleKey", "Toggle pixel doubling" },
 					{ "ToggleMuteKey", "Toggle audio mute" },
-					{ "TogglePlayerStanceColorsKey", "Toggle player stance colors" }
+					{ "TogglePlayerStanceColorKey", "Toggle player stance colors" },
```

Comment:
nit: Renaming Colors to Color doesn't look like an improvement (I assumed you will change all to Colors), because we have Colors in hotkey description, in Game.UsePlayerStanceColors property and after this renaming custom user hotkey in settings.yaml will be broken.

Reply:
This is only toggling one thing, so the "s" is inconsistent.  These changes have already broken half of the hotkeys in settings.yaml, and will break the rest when they move to mod-specific settings.